src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml

```diff
@@ -162,5 +162,14 @@
                 </dd>
             </dl>
         }
+
+        <form method="post">
+            <input asp-for="@Model.Name" type="hidden">
+            <input asp-for="@Model.ServiceId" type="hidden">
+            <button type="submit" class="govuk-button">
```

Comment:
This should have `data-module="govuk-button"`.
For more details see the [component page](https://design-system.service.gov.uk/components/button/
) and what it [does with it](https://github.com/alphagov/govuk-frontend/blob/c9ca0510c4c243875f99c64e3c0f215ccd26aa01/src/govuk/components/button/button.mjs).